scripts/setup_controller.sh

```diff
@@ -30,40 +30,41 @@ set -ex
-    echo $glusterVolume        >> /tmp/vars.txt
-    echo $siteFQDN             >> /tmp/vars.txt
-    echo $httpsTermination     >> /tmp/vars.txt
-    echo $dbIP                 >> /tmp/vars.txt
-    echo $adminpass            >> /tmp/vars.txt
-    echo $dbadminlogin         >> /tmp/vars.txt
-    echo $dbadminloginazure    >> /tmp/vars.txt
-    echo $dbadminpass          >> /tmp/vars.txt
-    echo $storageAccountName   >> /tmp/vars.txt
-    echo $storageAccountKey    >> /tmp/vars.txt
-    echo $redisDeploySwitch    >> /tmp/vars.txt
-    echo $redisDns             >> /tmp/vars.txt
-    echo $redisAuth            >> /tmp/vars.txt
-    echo $dbServerType                >> /tmp/vars.txt
-    echo $fileServerType              >> /tmp/vars.txt
-    echo $mssqlDbServiceObjectiveName >> /tmp/vars.txt
-    echo $mssqlDbEdition	>> /tmp/vars.txt
-    echo $mssqlDbSize	>> /tmp/vars.txt
-    echo $thumbprintSslCert >> /tmp/vars.txt
-    echo $thumbprintCaCert >> /tmp/vars.txt
-    echo $nfsByoIpExportPath >> /tmp/vars.txt
-    echo $phpVersion >> /tmp/vars.txt
-    echo $cmsApplication    >>/tmp/vars.txt
-    echo $lbDns             >>/tmp/vars.txt
-    echo $applicationDbName >>/tmp/vars.txt
-    echo $wpAdminPass       >>/tmp/vars.txt
-    echo $wpDbUserPass      >>/tmp/vars.txt
-    echo $wpVersion         >>/tmp/vars.txt
-    echo $sshUsername       >>/tmp/vars.txt
-    echo $storageAccountType >>/tmp/vars.txt
-    echo $fileServerDiskSize >>/tmp/vars.txt
-    echo $frontDoorFQDN >> /tmp/vars.txt
-    echo $azureFileShareType >>/tmp/vars.txt
+    echo "glusterNode                   = $glusterNode"                     >> /tmp/vars.txt
+    echo "glusterVolume                 = $glusterVolume"                   >> /tmp/vars.txt
+    echo "siteFQDN                      = $siteFQDN"                        >> /tmp/vars.txt
+    echo "httpsTermination              = $httpsTermination"                >> /tmp/vars.txt
+    echo "dbIP                          = $dbIP"                            >> /tmp/vars.txt
+    echo "adminpass                     = $adminpass"                       >> /tmp/vars.txt
+    echo "dbadminlogin                  = $dbadminlogin"                    >> /tmp/vars.txt
+    echo "dbadminloginazure             = $dbadminloginazure"               >> /tmp/vars.txt
+    echo "dbadminpass                   = $dbadminpass"                     >> /tmp/vars.txt
+    echo "storageAccountName            = $storageAccountName"              >> /tmp/vars.txt
+    echo "storageAccountKey             = $storageAccountKey"               >> /tmp/vars.txt
+    echo "redisDeploySwitch             = $redisDeploySwitch"               >> /tmp/vars.txt
+    echo "redisDns                      = $redisDns"                        >> /tmp/vars.txt
+    echo "redisDnsPort                  = $redisDnsPort"                    >> /tmp/vars.txt
+    echo "redisPassword                 = $redisPassword"                   >> /tmp/vars.txt
```

Comment:
hope you verified in all places the variable is renamed from redisAuth to redisPassword.

Reply:
yes, i have. thanks